src/Tester/FSharpGrainTests.cs

```diff
@@ -14,20 +12,9 @@ namespace UnitTests.General
     /// Unit tests for grains implementing generic interfaces
     /// </summary>
     [TestClass]
-    public class FSharpGrainTests : UnitTestSiloHost
+    public class FSharpGrainTests : UnitTestSiloHostEnsureDefaultStartedSecondaryOff
```

Comment:
Would it be clearer if this was renamed to `UnitTestSiloHostEnsureSingleSiloStarted` ?